src/App.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
 import RecipeListing from "./pages/recipes/RecipeListing";
-import Login from "./pages/login/login";
+import Login from "./pages/login/Login";
```

Comment:
What do you think to change to the pascal case dirs names too?

Reply:
Adjusted in subsequent commits 👍